src/gui/addnewtorrentdialog.ui

```diff
@@ -6,332 +6,979 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>414</width>
-    <height>630</height>
+    <width>936</width>
+    <height>500</height>
    </rect>
   </property>
+  <property name="sizePolicy">
+   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
+    <horstretch>0</horstretch>
+    <verstretch>0</verstretch>
+   </sizepolicy>
+  </property>
+  <property name="minimumSize">
+   <size>
+    <width>936</width>
+    <height>500</height>
+   </size>
+  </property>
+  <property name="maximumSize">
+   <size>
+    <width>16777215</width>
```

Comment:
I don't have much time to look into the changes, however you should reset these (**and all other places**) values if it is not different from the default value, there is a button on the right side of qt creator/designer that resets its value.

Reply:
thx, fixed